conf/base_config.yaml

```diff
@@ -120,7 +140,7 @@ generate:
 training:
     name: null
     version: null
-    n_epochs: 50
+    n_epochs: 30
```

Comment:
Probably better to exclude changes to `base_config.yaml` from this PR unless they are required for multiband decoding